lab_1_classify_by_unigrams/main.py

```diff
@@ -37,32 +63,75 @@ def calculate_mse(predicted: list, actual: list) -> float | None:
     :param actual: a list of actual values
     :return: the score
     """
+    if isinstance(predicted, list) and isinstance(actual, list) and len(predicted) == len(actual):
+        sum_difference = 0
+        for i, value in enumerate(predicted):
```

Comment:
Instead of using enumerate and getting by index in the next line, try to use `zip`